python/graphframes/tests.py

```diff
@@ -15,63 +15,72 @@
 # limitations under the License.
 #
 
-import sys
+import os
 import tempfile
 import shutil
 import re
 
-if sys.version_info[:2] <= (2, 6):
-    try:
-        import unittest2 as unittest
-    except ImportError:
-        sys.stderr.write('Please install unittest2 to test with Python 2.6 or earlier')
-        sys.exit(1)
-else:
-    import unittest
-
-from pyspark import SparkContext
-from pyspark.sql import functions as sqlfunctions, SparkSession
+import pytest
+from pyspark import SparkConf, SparkContext
```

Comment:
make one PR where you update tests.. 
are you sure that we should be using pytest and not unittests? 

Apache spark are using unittest.